synthea-hiv/uploader/bundle.py

```diff
@@ -28,7 +28,7 @@
 class Bundle:
   """Object to be uploaded to FHIR server."""
 
-  def __init__(self, file_name: pathlib.PosixPath, bundle_dict: Dict[str, str]):
+  def __init__(self, file_name: pathlib.PosixPath, bundle_dict: Dict[str, Any]):
```

Comment:
In the `list_all_files` file, the type was changed from `pathlib.PosixPath` to `pathlib.Path`. Do we need to do the same here?

Reply:
Done.